sdk/Lusid.Sdk.Tests/LusidApiFactoryTests.cs

```diff
@@ -523,5 +523,32 @@ public void ApiFactoryHasNoDefaultHeadersWhenNoneAreSpecified()
             // assert
             Assert.IsEmpty(api.Configuration.DefaultHeaders);
         }
+
+        [Test]
+        public void ApiFactoryWithTimeout() 
+        {
+
+            int timeout = 200;
+            int defaultTimeout = 100000;
+
+            var factory = LusidApiFactoryBuilder.Build("secrets.json", timeout);
+
+            var api = factory.Api<ScopesApi>();
+            DateTime start = DateTime.Now;
+
+            Assert.That(() => api.ListScopes(null), Throws.InstanceOf<ApiException>().With.Message.EqualTo("Internal SDK error occurred when calling ListScopes: The operation has timed out."));
+
+            try
+            {
```

Comment:
do you need to call the api again?

is it sufficient to just record the end time after the first assert and then run your second assert?